_ensure_reactive_charms_master_libraries

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash -e
+#
+# Ensure that charm-helpers and charms.openstack are from the master branch in
+# git.
+
+charms="$(cat charms.txt)"
+basedir="$(pwd)"
+username="$2"
+release="$1"
+usage="usage: _ensure_reactive_charms_master_libraries"
```

Comment:
missing arguments?

Reply:
copy/n/paste error; it doesn't use username or release; I'll update it.